test/expected.txt

```diff
@@ -905,6 +905,7 @@ PASS	/scroll-animations/view-timelines/block-view-timeline-current-time.tentativ
 FAIL	/scroll-animations/view-timelines/block-view-timeline-nested-subject.tentative.html	View timeline with subject that is not a direct descendant of the scroll container
 FAIL	/scroll-animations/view-timelines/change-animation-range-updates-play-state.html	Changing the animation range updates the play state
 FAIL	/scroll-animations/view-timelines/contain-alignment.html	Stability of animated elements aligned to the bounds of a contain region
+PASS	/scroll-animations/view-timelines/fieldset-source.html	Fieldset is a valid source for a view timeline
```

Comment:
This test is flaky: it always passes for me locally but never when ran via the GitHub actions.

Reply:
I just assumed it was a newly added test 🙈. 

It seems to have passed when ran via the GitHub actions now, or am I missing something?
https://github.com/flackr/scroll-timeline/actions/runs/7602264720/job/20702607225?pr=206#step:9:11

Reply:
O wait, I’m wrong here. A run earlier today did include it. Ignore my original comment here.